components/director/internal/statusupdate/handler_test.go

```diff
@@ -0,0 +1,226 @@
+package statusupdate_test
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/mock"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/kyma-incubator/compass/components/director/internal/statusupdate"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kyma-incubator/compass/components/director/internal/statusupdate/automock"
+	persistenceautomock "github.com/kyma-incubator/compass/components/director/pkg/persistence/automock"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/persistence/txtest"
+
+	"github.com/kyma-incubator/compass/components/director/internal/consumer"
+)
+
+const (
+	applicationsTable = "applications"
+	runtimesTable     = "runtimes"
+)
+
+func TestUpdate_Handler(t *testing.T) {
+	//given
+	testErr := errors.New("test")
+	txGen := txtest.NewTransactionContextGenerator(testErr)
+
+	testCases := []struct {
+		Name             string
+		TxFn             func() (*persistenceautomock.PersistenceTx, *persistenceautomock.Transactioner)
+		RepoFn           func() *automock.StatusUpdateRepository
+		Request          *http.Request
+		ExpectedStatus   int
+		ExpectedResponse string
+		MockHandler      http.Handler
+	}{
+		{
+			Name: "Success when integration system",
```

Comment:
what does it mean success? If you name this test, like "In case of Integration System, do nothing and execute next handler"  I would learn more just from the test name about the functionality.